### PR TITLE
DR | Move primary phone code to shared

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > As a first step to adding a primary phone question to HLR & NOD, we need to move the Supplemental Claim primary phone question code to the shared folder
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
  > Breaking up the main work into smaller PRs
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#68869](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68869)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

N/A - No visual change

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by powellkerry — DISMISSED** [review]

**Review by anniebtran — COMMENTED** [review]
just a quick question about some imports but otherwise looking good!

File: src/applications/appeals/shared/tests/components/PrimaryPhone.unit.spec.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 } from '@department-of-veterans-affairs/platform-forms-system/ui';
 
 import { PrimaryPhone } from '../../components/PrimaryPhone';
-import { PRIMARY_PHONE, errorMessages } from '../../constants';
+import { PRIMARY_PHONE, errorMessages } from '../../../995/constants';
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/shared/tests/components/PrimaryPhone.unit.spec.jsx:12]: Good catch! Yeah, `PRIMARY_PHONE` is in shared constants and `errorMessages` is in the content folder

File: src/applications/appeals/shared/tests/components/PrimaryPhoneReview.unit.spec.jsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import sinon from 'sinon';
 import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
 
 import PrimaryPhoneReview from '../../components/PrimaryPhoneReview';
-import { PRIMARY_PHONE, errorMessages } from '../../constants';
+import { PRIMARY_PHONE, errorMessages } from '../../../995/constants';
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/shared/tests/components/PrimaryPhoneReview.unit.spec.jsx:9]: Same situation here!

[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/shared/tests/components/PrimaryPhoneReview.unit.spec.jsx]: Oh boy, I've found and fixed a few more!

**Review by anniebtran — DISMISSED** [review]
LGTM

**Pull request closed without merging**